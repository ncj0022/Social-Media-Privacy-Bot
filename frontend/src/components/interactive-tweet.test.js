import React from 'react';
import renderer from 'react-test-renderer';

import InteractiveTweet from './interactive-tweet.component';

it('renders correctly', () => {
    
    const component = renderer.create(<InteractiveTweet />).toJSON();
    expect(component).toMatchSnapshot();   
});
