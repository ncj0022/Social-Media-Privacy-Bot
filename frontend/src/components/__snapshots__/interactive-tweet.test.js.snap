// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <textarea
      cols="100"
      placeholder="Example Tweet"
      readOnly={true}
      rows="8"
      value=""
    />
  </div>
  <div
    className="btn-group mr-2"
    role="group"
  >
    <button
      className="btn btn-danger"
      onClick={[Function]}
    >
       SPAM 
    </button>
    <button
      className="btn btn-success"
      onClick={[Function]}
    >
       HAM 
    </button>
  </div>
  <div
    className="btn-group"
    role="group"
  >
    <button
      className="btn btn-secondary"
      onClick={[Function]}
    >
       NEXT 
    </button>
  </div>
</div>
`;
